static/js/components/tiles/ranking_tile.tsx

```diff
@@ -182,15 +192,28 @@ export function RankingTile(props: RankingTilePropType): JSX.Element {
     </div>
   );
 
-  function handleEmbed(rankingPoints: RankingPoint[]): void {
+  function handleEmbed(
+    rankingPoints: RankingPoint[],
+    sources: Set<string>
+  ): void {
+    let chartHtml = "";
+    let chartHeight = 0;
+    if (rankingUnitHighestContainer.current) {
```

Comment:
i htink just checking highest and lowest probably is not sufficient? because user can be trying to export the lowest ranking unit, but there also is a highest ranking unit present. We probably want an id associated with each ranking unit so we can specify which one is being exported

Reply:
done